docs/tiers.md

```diff
@@ -0,0 +1,487 @@
+# Configuring Subscription Tiers
+
+This guide explains how to configure subscription tiers for model access control, 
+including tier-to-group mappings, authorization policies, and rate limiting. 
+
+Tiers enable differentiated service levels (e.g., free, premium, enterprise) with varying access permissions and quotas.
+
+> [!NOTE]
+> **For token management**: This guide covers tier configuration from an operator perspective. 
+> For information on how users obtain and use tokens, see [Token Management](token-management.md).
+
+---
+
+## Table of Contents
+
+1. [Prerequisites](#prerequisites)
+1. [Concepts Overview](#concepts-overview)
+1. [Architecture: How Tiering Works](#architecture-how-tiering-works)
+1. [Configuration Steps](#configuration-steps)
+1. [Verifying Your Configuration](#verifying-your-configuration)
+1. [Common Configuration Mistakes](#common-configuration-mistakes)
+1. [Related Documentation](#related-documentation)
+
+---
+
+## Prerequisites
+
+Before configuring tiers, ensure you have:
+
+- **Openshift cluster** with Gateway API support
+- **Kuadrant** for authentication, authorization and rate-limiting policies
+- **maas-api service** deployed and operational
+- **`kubectl` access** with cluster-admin or equivalent permissions
+
+**Core concepts:**
+- Kubernetes Role-Based Access Control (RBAC)
+- Kubernetes Service Accounts and bound tokens
+- Kubernetes TokenReview and SubjectAccessReview (SAR) APIs
+- Gateway API resources
+- Basic understanding of JWT authentication
+
+---
+
+## Concepts Overview
+
+### Core Terminology
+
+#### Tier  
+A subscription level that determines class of service (e.g., `free`, `premium`, `enterprise`). 
```

Comment:
Thought I remember small, medium large tshirt sizing, good with any of them just in case this was an autogen!

Reply:
I am well aware, but I do not think we have a decision about naming yet.